plugins/huly-lang-configurator/resources/META-INF/plugin.xml

```diff
@@ -0,0 +1,24 @@
+<!-- Copyright © 2024 Hardcore Engineering Inc. Use of this source code is governed by the Apache 2.0 license. -->
+
+<idea-plugin implementation-detail="true">
+  <name>Huly Code Lang Configurator</name>
+  <category>Other Tools</category>
+  <id>hulylabs.hulycode.plugins.hulylangconfigurator</id>
+  <vendor>JetBrains</vendor>
```

Comment:
We are not jetbrains, maybe `<vendor url="https://hulylabs.com/">Huly Labs</vendor>`